spec/support/user_helpers.rb

```diff
@@ -0,0 +1,7 @@
+module UserHelpers
+
+  def stub_current_user_uuid(uuid)
+    allow_any_instance_of(ApplicationController).to receive(:current_user_uuid) { uuid }
```

Comment:
allow_any_instance_of(ApplicationController).to receive(: current_user_uuid).and_return(: uuid)?  seems just a tad bit more intention revealing.  If there's a good reason to use the block form, would like to know that tho.

Reply:
I can change this.  I normally go with the block form to make shorter lines :-)